src/metabase/api/database.clj

```diff
@@ -478,18 +530,21 @@
   Tables are returned in the format `[table_name \"Table\"]`;
   When Fields have a semantic_type, they are returned in the format `[field_name \"table_name base_type semantic_type\"]`
   When Fields lack a semantic_type, they are returned in the format `[field_name \"table_name base_type\"]`"
```

Comment:
I'll think we'll appreciate a document for the new parameter here.

These docs will be automatically generated and published on our website, for example [doc for this api](https://www.metabase.com/docs/latest/api/database)

We currently don't have a complete document for all APIs, but I think we should start with documenting things that we're adding.